home/tests.py

```diff
@@ -165,22 +92,16 @@ def question_test_data():
 
 
 @pytest.fixture
-def tag_test_data():
-    tag = Tag(tag_name='test_tag_1')
-    tag.save()
-    return tag
+def profile():
+    profile = Profile.create(username="test_user", password="testtest", email="test@test.com")
+    return profile
```

Comment:
Great job,
After PR #65 will be merged you can replace these lines with `return request.profile`